protocol/contracts/libraries/Curve/LibCurve.sol

```diff
@@ -37,7 +43,7 @@ library LibCurve {
         uint256 x = xp[i] + ((1 * rates[i]) / PRECISION);
         uint256 y = getY(x, xp, a, D);
         uint256 dy = xp[j] - y - 1;
-        return dy / 1e6;
+        return dy / 1e6; // !
```

Comment:
no clue

Reply:
I flagged it during my docs review because it's a floating constant which I believe is specific to a particular pool composition of tokens (i.e. if this library was used for tokens with other decimals, this may fail)